src/tokens/solana.tokenlist.json

```diff
@@ -14059,11 +14059,20 @@
         "twitter": "https://twitter.com/polyplaycoin",
         "discord": "https://discord.gg/TAgndUz2Fc"
       }
+    },
+    {
+      "chainId": 101,
+      "address": "J5gLhk6mmQ4PSoir1Ufh8JY2ytEHA93YupzYiTFVCgcL",
+      "symbol": "FAROUT",
+      "name": "Far-Out Token",
+      "decimals": 8,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/assets/J5gLhk6mmQ4PSoir1Ufh8JY2ytEHA93YupzYiTFVCgcL/logo.png",
+      "tags": []
```

Comment:
please add links to social and website.

Reply:
Updated what I can today. Was not expecting to be required to supply these information at this point.